pandas/core/groupby/generic.py

```diff
@@ -1004,15 +1010,19 @@ def _aggregate_named(self, func, *args, **kwargs):
     @Substitution(klass="Series", selected="A.")
     @Appender(_transform_template)
     def transform(self, func, *args, **kwargs):
-        func = self._is_cython_func(func) or func
+        func = self._get_cython_func(func) or func
 
-        # if string function
         if isinstance(func, str):
-            if func in base.cython_transforms:
-                # cythonized transform
+            if not (func in base.transform_recognized_functions):
```

Comment:
a point on naming, I actually like _kernels, so would consider renaming the _functions -> _kernels